test-resources/test-harness/lambdas/src/dequeue-handler.ts

```diff
@@ -0,0 +1,55 @@
+import { PutItemCommand, DynamoDBClient } from "@aws-sdk/client-dynamodb";
+import type { LambdaInterface } from "@aws-lambda-powertools/commons/types";
+import { Logger } from "@aws-lambda-powertools/logger";
+import { SQSEvent, SQSBatchItemFailure } from "aws-lambda";
+
+export const logger = new Logger();
+export const dbClient = new DynamoDBClient({ region: process.env.REGION });
+
+export class DequeueLambdaHandler implements LambdaInterface {
+    async handler(event: SQSEvent): Promise<{ batchItemFailures: SQSBatchItemFailure[] }> {
+        logger.info("Starting to process records");
+
+        const tableName = process.env.EVENTS_TABLE_NAME;
+        const batchItemFailures: SQSBatchItemFailure[] = [];
+
+        for await (const { body, messageId } of event.Records) {
```

Comment:
await not sure it's need here  think it this one that's important `await dbClient.send(putItemCommand);`